guide/english/certifications/data-visualization/data-visualization-with-d3/work-with-dynamic-data-in-d3/index.md

```diff
@@ -8,3 +8,13 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 <a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
 
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+
+selection.text((d) => d);
+
+<a href="https://medium.com/@EleftheriaBatsou/free-code-camp-data-visualization-3d0bccae171f">Click here for the full guide.</a>
+
+```javascript
+.text((d) => (d + “ ETH”))
+```
+
+[![Video Walkthrough](http://img.youtube.com/vi/Lm3ll3M5ANo/0.jpg)](http://www.youtube.com/watch?v=Lm3ll3M5ANo "D3.JS -- Changing style, adding classes | FreeCodeCamp Challenges")
```

Comment:
We don't support adding links to video or resources which might be removed by a user not under freeCodeCamp's control